crates/lib/src/qpu/api.rs

```diff
@@ -356,6 +326,55 @@ impl ExecutionOptions {
     }
 }
 
+#[cached(
+    result = true,
+    time = 60,
```

Comment:
Why a TTL? Do we expect the accessor to change?

Reply:
I threw that out arbitrarily when I put together this draft, I think accessors would very rarely change, if at all. I suppose a reasonable TTL would be a minimum duration we would want to maintain an old accessor after putting up a new one, that way users don't experience a disruption when we swap them out.